src/auth/signup/components/signUp.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import signUpUser from '../redux/actions/signUpAction';
+import validation from '../validation';
+import '../CSS/signup.scss';
+
+const Loader = require('react-loader');
+
+export const SignUpUserForm = ({ signUpUser: signUp, isLoading }) => {
+  const [fields, setFields] = useState({ username: '', email: '', password: '' });
+  const [touched, setTouched] = useState({ username: false, email: false, password: false });
+
+  const onSubmit = (e) => {
+    const post = {
```

Comment:
Use a more meaningful name for this object, something like `userData` would suffice?

Reply:
okay,
I've done that